crd-generator/api/src/test/java/io/fabric8/crd/example/joke/JokeRequestSpec.java

```diff
@@ -42,7 +42,7 @@ public enum ExcludedTopic {
   @PrinterColumn(name = "jokeCategory", priority = 1)
   @JsonPropertyDescription("category-description")
   private Category category = Category.Any;
-  @PrinterColumn(name = "excludedTopics")
+  @io.fabric8.kubernetes.model.annotation.PrinterColumn(name = "excludedTopics")
```

Comment:
I think you're still using deprecated `PrinterColumn` here.

Reply:
+1. I think imports in the `io.fabric8.crd.generator.zookeeper.v1.ZookeeperStatus` also must be updated because it still uses deprecated version of `PrinterColumn`.

Reply:
Yes, that's intentional, we want to test that both annotations will work.

Reply:
then when do you plan to replace old annotations?

Reply:
With the release of the next Minor, theoretically 6.9 at this point.